bin/unset-maintenance-mode

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+ENVIRONMENT_NAME=$1
+
+case $ENVIRONMENT_NAME in
+  "development")
+    SUBSCRIPTION_ID="8655985a-2f87-44d7-a541-0be9a8c2779d"
+    RESOURCE_GROUP="s118d01-app"
+    APP_SERVICE="$RESOURCE_GROUP-as"
+    ;;
+  "production")
+    SUBSCRIPTION_ID="88bd392f-df19-458b-a100-22b4429060ed"
+    RESOURCE_GROUP="s118p01-app"
+    APP_SERVICE="$RESOURCE_GROUP-as"
+    ;;
+  *)
+    echo "Could not find a known environment with the name: $ENVIRONMENT_NAME"
+    exit 1
+    ;;
+esac
+
+if ! az account show > /dev/null; then
+  echo "Logging in..."
+  az login
+fi
+
+echo "Setting default subscription to $SUBSCRIPTION_ID..."
+az account set --subscription "$SUBSCRIPTION_ID"
+
+echo "Unsetting maintenance environment variables..."
+az webapp config appsettings set -g $RESOURCE_GROUP \
+      -n $APP_SERVICE \
+      --settings \
+      MAINTENANCE_MODE="" \
+      MAINTENANCE_MODE_AVAILABILITY_MESSAGE="" \
+      > /dev/null
```

Comment:
Indentation's a little odd at 6 spaces for these. Think we could make them 2 spaces?